### ISSUE TITLE
Should We Use PostgreSQL, MySQL, MongoDB or Supabase?

### ISSUE THREAD

**Issue by COSSAVVU** [COSSAVVU/CoSSAVVUDB#1]
# 👋 Greetings CoSSA
We need to build a Database for CoSSA for future purposes 
Options:
- MySQL
- PostgreSQL
- Supabase
- MongoDB
_What are Your Thoughts?_

**Comment by Attenang** [comment]
MySQL

**Comment by d-donne** [comment]
The question seems rather generic. What kind of application ideas do we have for the database.
The options are all relational databases.
I think Postgresql could scale up better.

**Comment by hyiaman** [comment]
I think that the stack we use will determine the kind of database, whether relational or non-relational. 
Are we set on a stack yet.?

**Comment by COSSAVVU** [comment]
> I think that the stack we use will determine the kind of database, whether relational or non-relational. Are we set on a stack yet.?

Yes, indeed we are assessing the options here. We can use the data for endless Possibilities.

**Comment by programmer-233** [comment]
### Short Answer: PostgreSQL
*Long Answer:* 
Hey Y'all I would go with Building all of them (for the experience) then Benchmark their performance. Choose the winner based on Latency, Scalability and Price of hosting.  Which *PostgreSQL* does very well!

*Considerations:* We're taught MySQL and Microsoft SQL Server. So choosing any DB that students would easily maintain it would be nice. I understand MYSQL fans for this case but again choosing something they're not taught in class would be the best for their learning. 
*Supabase* has a good free tier and has PostgreSQL so would be a great choice too.
